.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+# Pull Request Checklist
+
+- [ ] My code follows the [code style](https://github.com/asfadmin/ia-standards/blob/main/standards.md) requirements of this project.
+- [ ] Resources and Data Structures are sorted by ABC or a defined sorting pattern.
+- [ ] Code requirements are pinned.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing unit tests passed.
+- [ ] I have deployed my code.
+- [ ] All new and existing E2E/Integration tests passed.
```

Comment:
Not applicable, this is a library, there are no end to end tests and no deployment. Testing is done via unit tests.
```suggestion
```